examples/storage-examples/gluster-examples/nginx_gluster_plugin/README.md

```diff
@@ -0,0 +1,243 @@
+
+
+- For this example, you will have to define the GlusterFS Cluster as “EndPoints” within Kubernets/OpenShift platform using a file similar to [endpoints configuration file](gluster-endpoints.json).  You must define the IP/Hostname of your gluster servers and the port that you want to use.  The port value can be any numeric value within the accepted range of ports.
+
+
+        { 
+           "kind": "Endpoints", 
+           "apiVersion": "v1", 
+           "metadata": { 
+               "name": "glusterfs-cluster" 
+           }, 
+           "subsets": [ 
+               { 
+                   "addresses": [ 
+                      { 
+                          "IP": "192.168.122.221"
+                      } 
+                   ], 
+                   "ports": [ 
+                       { 
+                           "port": 1 
+                       } 
+                   ] 
+               }, 
+               { 
+                   "addresses": [ 
+                       { 
+                            "IP": "192.168.122.222"
+                       } 
+                   ], 
+                   "ports": [ 
+                       { 
+                            "port": 1 
+                       } 
+                   ] 
+               } 
+           ] 
+        } 
+
+
+*Note: It seems that the IP value under `addresses` must be an actual IP Address of the gluster server, not a fully qualified hostname, this will most likely change in future releases.*
+
+
+
+        oc create -f gluster-endpoints.json
+
+        [root@OpenShift1 nginx_gluster]# oc create -f gluster-endpoints.json 
+        endpoints/glusterfs-cluster 
+
+
+        oc get endpoints
+
+        [root@OpenShift1 nginx_gluster]# oc get endpoints 
+        NAME                ENDPOINTS 
+        glusterfs-cluster   192.168.122.221:1,192.168.122.222:1 
+        kubernetes          192.168.122.251:8443 
+
+
+
+
+- Following similar deployment to what we did in the previous examples, we will modify the [pod configuration file](gluster-nginx-pod.json) to utilize the “glusterfs” plugin
+
+
+        { 
+            "apiVersion": "v1", 
+            "id": "glusterfs-nginx", 
+            "kind": "Pod", 
+            "metadata": { 
+                "name": "glusterfs-nginx" 
+            }, 
+            "spec": { 
+                "containers": [ 
+                    { 
+                        "name": "glusterfs-nginx", 
+                        "image": "fedora/nginx", 
+                        "volumeMounts": [ 
+                            { 
+                                 "mountPath": "/usr/share/nginx/html/test", 
+                                 "name": "nginxglustervol" 
+                            } 
+                        ] ,
+                        "securityContext": { 
+                             "capabilities": {}, 
+                             "privileged": true 
```

Comment:
@screeley44 @wattsteve said you verified that privileged: true no longer required?